data/json/mapgen/crater.json

```diff
@@ -1,4 +1,15 @@
 [
+  {
+    "type": "palette",
+    "id": "crater_palette",
+    "terrain": {
+      ".": "t_region_groundcover_barren",
+      "?": [ [ "t_region_groundcover_barren", 3 ], "t_null" ],
+      "0": "t_region_groundcover_barren"
+    },
+    "items": { ".": { "item": "wreckage", "chance": 2 }, "?": { "item": "wreckage", "chance": 1 } },
+    "nested": { ".": { "chunks": [ [ "rubble", 10 ], [ "null", 3 ] ] }, "?": { "chunks": [ [ "rubble", 3 ], [ "null", 10 ] ] } }
```

Comment:
What's the point to create 1x1 nests, if you can simply 
```json
"furniture": {
      ".": [ [ "f_rubble", 10 ], [ "null", 3 ] ]
      "?": [ [ "f_rubble", 3 ], [ "null", 10 ] ]
    },
```

Reply:
place_rubble bashes terrain/furniture which is somewhat relevant bc it's using fallback_predecessor_mapgen so there could be road/trees/street lamps etc on the ?, the current way . is set up that stuff would've already been dismantled so it'd be useless there but I might still tweak the terrain distributions anyway I'm getting the shapes how I want first

Reply:
There is also this #66269 which might eventually lead to rubble and set bash being more appropriate than manually placing rubble for general use too